service/insolvency_service.go

```diff
@@ -78,6 +78,18 @@ func ValidateInsolvencyDetails(svc dao.Service, transactionID string) (bool, *[]
 		}
 	}
 
+	// Check if attachments are present, if false then at least one appointed practitioner must be present
+	hasAttachments := true
+	if len(insolvencyResource.Data.Attachments) == 0 {
+		hasAttachments = false
+	}
+	if !hasAttachments && !hasAppointedPractitioner {
+		validationError := fmt.Sprintf("error - at least one practitioner must be appointed if there are no attachments for insolvency case with transaction id [%s]", insolvencyResource.TransactionID)
```

Comment:
Would make a small change to the error message. Change `if there are no attachments` to `as there are no attachments` as we already know there are no attachments added so it's more of an instruction